docs/docs/node-apis.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: Gatsby Node APIs
-description: Documentation on Node APIs used in Gatsby build process for common uses like creating pages
+title: Gatsby নোড এপিআই|
+description: পেট তৈরির মতো সাধারণ ব্যবহারের জন্য Gatsby বিল্ড প্রক্রিয়াতে ব্যবহৃত নোড এপিআইগুলিতে ডকুমেন্টেশন|
 jsdoc: ["gatsby/src/utils/api-node-docs.js"]
 apiCalls: NodeAPI
 ---
 
-Gatsby gives plugins and site builders many APIs for controlling your site's data in the GraphQL data layer.
+Gatsby গ্রাফকিউল ডেটা স্তরে আপনার সাইটের ডেটা নিয়ন্ত্রণের জন্য প্লাগইন এবং সাইট নির্মাতাকে অনেকগুলি এপিআই দেয়|
 
 ## Async plugins
 
-If your plugin performs async operations (disk I/O, database access, calling remote APIs, etc.) you must either return a promise or use the callback passed to the 3rd argument. Gatsby needs to know when plugins are finished as some APIs, to work correctly, require previous APIs to be complete first. See [Debugging Async Lifecycles](/docs/debugging-async-lifecycles/) for more info.
+যদি আপনার প্লাগইন অ্যাসিঙ্ক অপারেশন করে (disk I/O, database access, calling remote APIs, etc.) আপনাকে হয় কোনও প্রতিশ্রুতি ফিরিয়ে দিতে হবে বা তৃতীয় যুক্তির সাথে প্রেরিত কলব্যাকটি ব্যবহার করতে হবে। Gatsby প্লাগইনগুলি কিছু API হিসাবে শেষ হয়ে গেলে, সঠিকভাবে কাজ করার জন্য, পূর্ববর্তী API গুলি প্রথমে সম্পূর্ণ হওয়া দরকার জানা| দেখা [Debugging Async Lifecycles](/docs/debugging-async-lifecycles/) আরও তথ্যের জন্য.
```

Comment:
প্রতিশ্রুতি?
তৃতীয় যুক্তির?
Can you please re-translate the entire paragraph?

Reply:
I think entire paragraph is correct.